### PR TITLE
add cdsapi package to access Copernicus Climate Data Store

### PR DESCRIPTION
**Pull request by StephanSiemen**

### CODE REVIEW

**Comment by StephanSiemen** [comment]
Now failing on Azure Windows test - common problem see #7715

**Comment by StephanSiemen** [comment]
@conda-forge/core - this PR should be ready to merge. The only test failing seems to be a general Azure problem (see #7715). Thanks

File: recipes/cdsapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/82/2b/51601be330f4d27d71b247b83cee1967139b12175ebb94a51f9eb5b52ee2/cdsapi-0.1.3.tar.gz
+  sha256: 9db05f66bcc8385742aa389768f892f813ebbc033427c25f45dc6d89445bd08f
+
+build:
+  number: 10
```
Comment (scopatz) [inline recipes/cdsapi/meta.yaml]: This should be zero in staged-recipes.

File: recipes/cdsapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/82/2b/51601be330f4d27d71b247b83cee1967139b12175ebb94a51f9eb5b52ee2/cdsapi-0.1.3.tar.gz
+  sha256: 9db05f66bcc8385742aa389768f892f813ebbc033427c25f45dc6d89445bd08f
+
+build:
```
Comment (scopatz) [inline recipes/cdsapi/meta.yaml]: Can this be `noarch: python`?

Comment (StephanSiemen) [reply recipes/cdsapi/meta.yaml]: Yes, it can. Have added it

File: recipes/cdsapi/meta.yaml (hunk 1 above)
Comment (StephanSiemen) [reply recipes/cdsapi/meta.yaml]: Updated meta.yaml to say build number 0

**Comment by scopatz** [comment]
noarch only needs circle, merging

**Comment by StephanSiemen** [comment]
Thanks for help, @scopatz !

**Pull request merged**